web/src/main/java/org/bookkeepervisualmanager/api/resources/LedgerBean.java

```diff
@@ -47,7 +47,7 @@
     private boolean closed;
     private String state;
     private int metadataFormatVersion;
-    private List<String> bookies;
+    private Map<Long, List<String>> ensambles;
```

Comment:
Typo: Ensemble

How can you be sure that ensembles will be listed sorted by key?
Isn't  a List<EnsembleInfo> better?
Where EnsembleInfo is a structure with startEntryId and list of bookies

Reply:
Ok for typo.
To preserve both, original map key and ordering, i convert it in a linked hash map.